Gemfile

```diff
@@ -28,6 +28,8 @@ gem "rackup" if use_rackup
 
 gem "jruby-openssl", :platform => "jruby"
 
+gem "sqlite3", "~> 1.4"
```

Comment:
Add a comment on why we have this? Or make it conditional/opt-in? (ENV that the workflow can set) 

Very few run the turbo rails test when they clone puma repo and run tests, and as far as I know, you don't need SQLite then.

Reply:
CI currently appears to be broken, is it not important to fix that?